app/src/androidTest/java/com/github/houseorganizer/houseorganizer/GroceriesActivityTest.java

```diff
@@ -139,6 +138,24 @@ public void addingItemShowsNewItem() {
                         RecyclerViewHelper.clickChildViewWithId(R.id.delete_item_button)));
     }
 
+    @Test
+    public void modifyingItemModifiesIt(){
```

Comment:
For the other tests, does it matter if you don't change back the shop item's description, unit and name?

Reply:
Oh yes good catch !